app/assets/stylesheets/global/_extends.scss

```diff
@@ -0,0 +1,8 @@
+._clear {
```

Comment:
Why does this start with an underscore?

Reply:
Most languages use an underscore to mark privacy. If the value is written with an underscore, it shouldn't be used outside of an implementer. 

In this case, the implementer is Sass' `@extend` macro; the `._clear` class should never be used directly, such as `<div class="row _clear"></div>`.